packages/by-name/microsoft/genpolicy/package.nix

```diff
@@ -31,12 +31,16 @@ rustPlatform.buildRustPackage rec {
     patches = [
       # TODO(burgerdev): drop after Microsoft ported https://github.com/kata-containers/kata-containers/pull/9706
       (fetchpatch {
-        name = "genpolicy_device_support.patch";
+        name = "0001-genpolicy-add-rules-and-types-for-volumeDevices.patch";
```

Comment:
I'll take a note to vendor this patch, too.

Reply:
If we want that I can easily do it now. I thought it wound't be necessary to vendor it.

Reply:
Ah wait, the patch we are fetching isn't part of the upstream repo, got it.